src/core/Navigation/SideNavigation/SideNavigationItem/SideNavigationItem.tsx

```diff
@@ -7,19 +7,19 @@ import styled from 'styled-components';
 import { IconSubDirectory } from 'suomifi-icons';
 
 export interface SideNavigationItemProps {
-  /** Custom class */
+  /** Custom classname to extend or customise */
   className?: string;
-  /** Content for the link element. Use the polymorphic RouterLink component to get intended CSS styling */
+  /** Content of the link element. Use the polymorphic `<RouterLink>` component to get intended CSS styling */
```

Comment:
for

Reply:
This is now addressed in the latest commit